.github/workflows/release.yml

```diff
@@ -101,12 +101,12 @@ jobs:
           build-args: |
             "MASS_VERSION=${{ needs.build-and-publish-pypi.outputs.version }}"
 
-  #addon-version-update:
-  #  name: Server repo PR
-  #  needs: build-and-publish-pypi
-  #  runs-on: ubuntu-latest
-  #  steps:
-  #    - uses: music-assistant/addon-update-action@main
-  #      with:
-  #        github_token: ${{ secrets.PRIVILEGED_GITHUB_TOKEN }}
-  #        new_release_version: ${{ needs.build-and-publish-pypi.outputs.version }}
+  addon-version-update:
+    name: Server repo PR
+    needs: [ build-and-publish-pypi, build-and-push-container-image ]
```

Comment:
I think the `build-and-publish-pypi` bit is redundant and can be removed, since `build-and-push-container-image` already depends on it.

Reply:
I kept it in this case rather than adding a new output to the build-and-push-container-image job. I was tossing up what would be the best.
Let me double check what the default outputs are from that docker stuff

Reply:
From double checking a few sources, it seems that as I need the output created in the first job, I have to keep it as a dependency in the add-on update job.
It also feels cleaner to do this, rather than essentially read that output as an input in the docker build job and then output the same value again to use in the add-on update job.